common/src/main/java/com/wynntils/models/worlds/WorldStateModel.java

```diff
@@ -182,6 +182,9 @@ public void update(PlayerDisplayNameChangeEvent e) {
     private boolean setWorldIfMatched(Matcher m, boolean housing) {
         if (m.find()) {
             String worldName = housing ? currentWorldName : m.group(1);
+            if (worldName.isEmpty() && housing) {
+                WynntilsMod.warn("Changed world via housing join, current world name is unknown");
+            }
```

Comment:
Maybe set worldName to "Unknown Housing" instead of the empty string?

Reply:
World name should be "WC" followed by the number rather than the housing name, that is set below. So maybe "WC??" is better?

Reply:
Sure.